### DISCUSSION TITLE
Mobile not connecting to MM or Rainbow wallets

### DISCUSSION THREAD

**Discussion by codenamejason** [rainbow-me/rainbowkit#706] [General]
Mobile not connecting to MM or Rainbow wallets =>

When I connect on desktop everything works as expected. However when I use mobile iOS chrome or brave apps it just opens MetaMask and doesn't present the connect button. With 🌈 wallet it just spins and then says an error occurred.

The strange part is it works when using MetaMask browser 🤔

**Comment by karangoraniya** [comment]
Are you using Wagmi or Ethers Js to connect with Smart Contract ?
